boards/arm/mps2_an521/doc/index.rst

```diff
@@ -431,6 +424,8 @@ serial port:
 
    Hello World! mps2_an521
 
+.. note:: Please refer to the tfm_integration sample for more details about integrating with TF-M and multiple images scenario.
```

Comment:
While it's polite, the project preference is not to use "please", so start the sentence with ``Refer to the...``